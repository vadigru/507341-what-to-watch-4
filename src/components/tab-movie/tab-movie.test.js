import React from "react";
import renderer from "react-test-renderer";
import TabMovie from "../tab-movie/tab-movie.jsx";

const mockProps = {
  tabNames: [`Overview`, `Details`, `Reviews`],
  tabName: `tabName`,
  activeTab: `activeTab`,
  onTabClick: () => {}
};

it(`Should render TabMovie component`, () => {
  const tree = renderer
    .create(
        <TabMovie
          {...mockProps}
        />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
