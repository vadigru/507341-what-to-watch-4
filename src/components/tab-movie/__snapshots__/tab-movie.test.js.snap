// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render TabMovie component 1`] = `
<ul
  className="movie-nav__list"
>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Overview
    </a>
  </li>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Details
    </a>
  </li>
  <li
    className="movie-nav__item false"
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Reviews
    </a>
  </li>
</ul>
`;
